demoextendsymfonyform/composer.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "prestashop/demoextendsymfonyform",
```

Comment:
Maybe we should add a prefix 😄so we can create new ones later (or versions)

This one would be demoextendsymfonyform-2 as https://github.com/friends-of-presta/demo-cqrs-hooks-usage-module is kindof number one